README.md

```diff
@@ -6,9 +6,16 @@
 
 ![logo](https://raw.githubusercontent.com/opentok/accelerator-core/master/tokbox-logo.png)
 
-# OpenTok Accelerator Core
+# OpenTok Accelerator Core Web
 
+OpenTok's `Accelerator Core Web` provides a simple and effortless integration for OpenTok libraries. Wether you already built your application or are starting to use OpenTok's libraries, `Accelerator Core Web` gives you an easy and fast way to integrate audio/video communication into your application.
 
+##Quickstart
+
+ To start using basic audio/video communication with OpenTok's libraries go to [Basic Usage](#basic-usage). To implement a more granular and complex management for OpenTok's session you can check [Advanced OpenTok Session Management Using SDK-Wrapper](#advanced-opentok-session-management-using-sdk-wrapper). 
+ 
+ **The regular Initialization for `OpenTok Accelerator Core` is**
```

Comment:
This is not code.  I was just looking for a way to summarize the features of `core`.